src/styles/constants.css

```diff
@@ -8,6 +8,10 @@
   --page-footer-height: 2rem;
   --page-footer-height-mobile: 4.5rem;
 
+  /* Restriction Warning */
+  --restriction-warning-height: 3.25rem;
```

Comment:
feels a bit sketchy, how do we know this is true for all the zoom settings and browser sizes and user font settings and etc? That said, I suppose it's fine.

Reply:
user font settings will update base rems right?